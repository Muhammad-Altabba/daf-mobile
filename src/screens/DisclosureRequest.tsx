import * as React from 'react'
import {
  Container,
  Screen,
  Button,
  Constants,
  Banner,
  RequestItem,
} from '@kancha/kancha-ui'
import { NavigationScreenProps } from 'react-navigation'
import { Colors } from '../theme'

// tslint:disable-next-line:no-var-requires
const avatar1 = require('../assets/images/space-x-logo.jpg')

// tslint:disable-next-line:no-var-requires
const bannerImage = require('../assets/images/space-x-banner.jpg')

const Component: React.FC<NavigationScreenProps> = ({ navigation }) => {
  return (
    <Screen
      statusBarHidden={true}
      safeAreaBottom={true}
      safeAreaBottomBackground={Colors.WHITE}
      scrollEnabled={true}
      footerDivider={true}
      footerComponent={
        <Container
          paddingHorizontal={true}
          paddingBottom={true}
          backgroundColor={Colors.WHITE}
        >
          <Container flexDirection={'row'}>
            <Container flex={1}>
              <Button
                block={Constants.ButtonBlocks.Clear}
                type={Constants.BrandOptions.Warning}
                buttonText={'Decline'}
                onPress={() => navigation.goBack()}
              />
            </Container>
            <Container flex={1}>
              <Button
                block={Constants.ButtonBlocks.Filled}
                type={Constants.BrandOptions.Primary}
                buttonText={'Accept'}
                onPress={() => navigation.goBack()}
                shadowOpacity={0.2}
              />
            </Container>
          </Container>
        </Container>
      }
    >
      <Container>
        <Banner
          title={'Space X'}
          subTitle={'Blast off to the Moon'}
          avatar={avatar1}
          backgroundImage={bannerImage}
        />
        <Container>
          <RequestItem subTitle={'Name'}>Jack</RequestItem>
          <RequestItem subTitle={'Date of Birth'}>24-01-1991</RequestItem>
          <RequestItem subTitle={'Location'}>Ireland</RequestItem>
          <RequestItem subTitle={'Phone'} itemNote={'Mobile'}>
            555-876-882771
          </RequestItem>
        </Container>
      </Container>
    </Screen>
  )
}

export default Component
