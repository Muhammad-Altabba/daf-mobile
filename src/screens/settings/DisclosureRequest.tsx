import * as React from 'react'
import {
  Container,
  Screen,
  Button,
  Constants,
  Banner,
  RequestItem,
} from '@kancha/kancha-ui'
import { NavigationStackScreenProps } from 'react-navigation-stack'
import { Colors } from '../../theme'

// tslint:disable-next-line:no-var-requires
const avatar1 = require('../../assets/images/space-x-logo.jpg')

// tslint:disable-next-line:no-var-requires
const bannerImage = require('../../assets/images/space-x-banner.jpg')

const Component: React.FC<NavigationStackScreenProps> = ({ navigation }) => {
  return (
    <Screen
      safeAreaBottom={true}
      safeAreaBottomBackground={Colors.WHITE}
      scrollEnabled={true}
      footerDivider={true}
      footerComponent={
        <Container
          paddingHorizontal={true}
          paddingBottom={true}
          backgroundColor={Colors.WHITE}
        >
          <Container flexDirection={'row'}>
            <Container flex={1}>
              <Button
                block={Constants.ButtonBlocks.Clear}
                type={Constants.BrandOptions.Warning}
                buttonText={'Decline'}
                onPress={() => navigation.goBack()}
              />
            </Container>
            <Container flex={1}>
              <Button
                block={Constants.ButtonBlocks.Filled}
                type={Constants.BrandOptions.Primary}
                buttonText={'Accept'}
                onPress={() => navigation.goBack()}
                shadowOpacity={0.2}
              />
            </Container>
          </Container>
        </Container>
      }
    >
      <Container>
        <Banner
          title={'Space X'}
          subTitle={'Blast off to the Moon'}
          avatar={avatar1}
          backgroundImage={bannerImage}
        />
        <Container>
          <RequestItem
            subTitle={'Name'}
            options={[
              {
                id: '01',
                iss: 'None',
                property: 'name',
                value: 'Jack',
                selected: true,
              },
            ]}
          />
          <RequestItem
            subTitle={'Date of Birth'}
            options={[
              {
                id: '02',
                iss: 'None',
                property: 'dateOfBirth',
                value: '24-01-1991',
                selected: true,
              },
            ]}
          />
          <RequestItem
            subTitle={'Location'}
            options={[
              {
                id: '03',
                iss: 'None',
                property: 'location',
                value: 'Ireland',
                selected: true,
              },
            ]}
          />
          <RequestItem
            subTitle={'Phone'}
            itemNote={'Mobile'}
            options={[
              {
                id: '04',
                iss: 'None',
                property: 'phone',
                value: '+555-876-882771',
                selected: true,
              },
            ]}
          />
        </Container>
      </Container>
    </Screen>
  )
}

export default Component
