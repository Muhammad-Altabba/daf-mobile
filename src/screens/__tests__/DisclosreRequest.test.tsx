import 'react-native'
import React from 'react'
import DisclosureRequest from '../settings/DisclosureRequest'
import { render } from 'react-native-testing-library'

const navigation = {
  goBack: jest.fn(),
}

it('renders correctly', () => {
  // @ts-ignore
  const tree = render(<DisclosureRequest navigation={navigation} />).toJSON()
  expect(tree).toMatchSnapshot()
})
