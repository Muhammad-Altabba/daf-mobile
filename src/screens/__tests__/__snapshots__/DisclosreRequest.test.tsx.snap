// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <RCTScrollView
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    contentInsetAdjustmentBehavior="never"
    getTextInputRefs={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onScroll={[Function]}
    scrollEventThrottle={200}
    scrollToBottomOnKBShow={false}
    scrollToInputAdditionalOffset={75}
    startScrolledToBottom={false}
    style={
      Object {
        "backgroundColor": "#EFEEF3",
      }
    }
  >
    <View>
      <View
        pointerEvents="auto"
        style={
          Object {
            "alignItems": undefined,
            "backgroundColor": undefined,
            "borderRadius": undefined,
            "flex": 1,
            "flexDirection": undefined,
            "height": undefined,
            "justifyContent": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginTop": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingTop": undefined,
            "width": undefined,
          }
        }
      >
        <View
          pointerEvents="auto"
          style={
            Object {
              "alignItems": undefined,
              "backgroundColor": undefined,
              "borderRadius": undefined,
              "flex": undefined,
              "flexDirection": undefined,
              "height": undefined,
              "justifyContent": undefined,
              "margin": undefined,
              "marginBottom": undefined,
              "marginLeft": undefined,
              "marginRight": undefined,
              "marginTop": undefined,
              "padding": undefined,
              "paddingBottom": undefined,
              "paddingHorizontal": undefined,
              "paddingLeft": undefined,
              "paddingRight": undefined,
              "paddingTop": undefined,
              "width": undefined,
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "backgroundColor": "#333333",
                "flex": 1,
                "height": 250,
              }
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../src/assets/images/space-x-banner.jpg",
                }
              }
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": 250,
                    "width": undefined,
                  },
                  undefined,
                ]
              }
            />
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgba(0,0,0,0.3)",
                  "borderRadius": undefined,
                  "flex": 1,
                  "flexDirection": undefined,
                  "height": undefined,
                  "justifyContent": "center",
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": undefined,
                  "paddingHorizontal": undefined,
                  "paddingLeft": undefined,
                  "paddingRight": undefined,
                  "paddingTop": 15,
                  "width": undefined,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../src/assets/images/space-x-logo.jpg",
                  }
                }
                style={
                  Object {
                    "backgroundColor": "#0078ff",
                    "borderRadius": 5,
                    "height": 56,
                    "width": 56,
                  }
                }
              />
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": undefined,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": undefined,
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": 15,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#FFFFFF",
                      "fontSize": 24,
                      "fontWeight": "bold",
                    }
                  }
                >
                  Space X
                </Text>
                <View
                  pointerEvents="auto"
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": undefined,
                      "borderRadius": undefined,
                      "flex": undefined,
                      "flexDirection": undefined,
                      "height": undefined,
                      "justifyContent": undefined,
                      "margin": undefined,
                      "marginBottom": undefined,
                      "marginLeft": undefined,
                      "marginRight": undefined,
                      "marginTop": undefined,
                      "padding": undefined,
                      "paddingBottom": undefined,
                      "paddingHorizontal": undefined,
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "paddingTop": 5,
                      "width": undefined,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#FFFFFF",
                        "fontSize": 14,
                      }
                    }
                  >
                    Blast off to the Moon
                  </Text>
                </View>
              </View>
            </View>
          </View>
          <View
            pointerEvents="auto"
            style={
              Object {
                "alignItems": undefined,
                "backgroundColor": undefined,
                "borderRadius": undefined,
                "flex": undefined,
                "flexDirection": undefined,
                "height": undefined,
                "justifyContent": undefined,
                "margin": undefined,
                "marginBottom": undefined,
                "marginLeft": undefined,
                "marginRight": undefined,
                "marginTop": undefined,
                "padding": undefined,
                "paddingBottom": undefined,
                "paddingHorizontal": undefined,
                "paddingLeft": undefined,
                "paddingRight": undefined,
                "paddingTop": undefined,
                "width": undefined,
              }
            }
          >
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderBottomColor": "#CCCCCC",
                  "borderBottomWidth": 0.5,
                  "borderRadius": undefined,
                  "flex": undefined,
                  "flexDirection": "row",
                  "height": undefined,
                  "justifyContent": undefined,
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": 16,
                  "paddingHorizontal": undefined,
                  "paddingLeft": 24,
                  "paddingRight": undefined,
                  "paddingTop": 16,
                  "width": undefined,
                }
              }
            >
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": undefined,
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": 1,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": undefined,
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#BBBBBB",
                      "fontSize": 14,
                    }
                  }
                >
                  Name
                </Text>
                <View
                  pointerEvents="auto"
                  style={
                    Object {
                      "alignItems": undefined,
                      "backgroundColor": undefined,
                      "borderRadius": undefined,
                      "flex": undefined,
                      "flexDirection": undefined,
                      "height": undefined,
                      "justifyContent": undefined,
                      "margin": undefined,
                      "marginBottom": undefined,
                      "marginLeft": undefined,
                      "marginRight": undefined,
                      "marginTop": undefined,
                      "padding": undefined,
                      "paddingBottom": undefined,
                      "paddingHorizontal": undefined,
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "paddingTop": 2,
                      "width": undefined,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#333333",
                        "fontSize": 16,
                        "lineHeight": 22,
                      }
                    }
                  >
                    Jack
                  </Text>
                </View>
              </View>
            </View>
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderBottomColor": "#CCCCCC",
                  "borderBottomWidth": 0.5,
                  "borderRadius": undefined,
                  "flex": undefined,
                  "flexDirection": "row",
                  "height": undefined,
                  "justifyContent": undefined,
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": 16,
                  "paddingHorizontal": undefined,
                  "paddingLeft": 24,
                  "paddingRight": undefined,
                  "paddingTop": 16,
                  "width": undefined,
                }
              }
            >
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": undefined,
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": 1,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": undefined,
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#BBBBBB",
                      "fontSize": 14,
                    }
                  }
                >
                  Date of Birth
                </Text>
                <View
                  pointerEvents="auto"
                  style={
                    Object {
                      "alignItems": undefined,
                      "backgroundColor": undefined,
                      "borderRadius": undefined,
                      "flex": undefined,
                      "flexDirection": undefined,
                      "height": undefined,
                      "justifyContent": undefined,
                      "margin": undefined,
                      "marginBottom": undefined,
                      "marginLeft": undefined,
                      "marginRight": undefined,
                      "marginTop": undefined,
                      "padding": undefined,
                      "paddingBottom": undefined,
                      "paddingHorizontal": undefined,
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "paddingTop": 2,
                      "width": undefined,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#333333",
                        "fontSize": 16,
                        "lineHeight": 22,
                      }
                    }
                  >
                    24-01-1991
                  </Text>
                </View>
              </View>
            </View>
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderBottomColor": "#CCCCCC",
                  "borderBottomWidth": 0.5,
                  "borderRadius": undefined,
                  "flex": undefined,
                  "flexDirection": "row",
                  "height": undefined,
                  "justifyContent": undefined,
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": 16,
                  "paddingHorizontal": undefined,
                  "paddingLeft": 24,
                  "paddingRight": undefined,
                  "paddingTop": 16,
                  "width": undefined,
                }
              }
            >
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": undefined,
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": 1,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": undefined,
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#BBBBBB",
                      "fontSize": 14,
                    }
                  }
                >
                  Location
                </Text>
                <View
                  pointerEvents="auto"
                  style={
                    Object {
                      "alignItems": undefined,
                      "backgroundColor": undefined,
                      "borderRadius": undefined,
                      "flex": undefined,
                      "flexDirection": undefined,
                      "height": undefined,
                      "justifyContent": undefined,
                      "margin": undefined,
                      "marginBottom": undefined,
                      "marginLeft": undefined,
                      "marginRight": undefined,
                      "marginTop": undefined,
                      "padding": undefined,
                      "paddingBottom": undefined,
                      "paddingHorizontal": undefined,
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "paddingTop": 2,
                      "width": undefined,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#333333",
                        "fontSize": 16,
                        "lineHeight": 22,
                      }
                    }
                  >
                    Ireland
                  </Text>
                </View>
              </View>
            </View>
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderBottomColor": "#CCCCCC",
                  "borderBottomWidth": 0.5,
                  "borderRadius": undefined,
                  "flex": undefined,
                  "flexDirection": "row",
                  "height": undefined,
                  "justifyContent": undefined,
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": 16,
                  "paddingHorizontal": undefined,
                  "paddingLeft": 24,
                  "paddingRight": undefined,
                  "paddingTop": 16,
                  "width": undefined,
                }
              }
            >
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": undefined,
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": 1,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": undefined,
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#BBBBBB",
                      "fontSize": 14,
                    }
                  }
                >
                  Phone
                </Text>
                <View
                  pointerEvents="auto"
                  style={
                    Object {
                      "alignItems": undefined,
                      "backgroundColor": undefined,
                      "borderRadius": undefined,
                      "flex": undefined,
                      "flexDirection": undefined,
                      "height": undefined,
                      "justifyContent": undefined,
                      "margin": undefined,
                      "marginBottom": undefined,
                      "marginLeft": undefined,
                      "marginRight": undefined,
                      "marginTop": undefined,
                      "padding": undefined,
                      "paddingBottom": undefined,
                      "paddingHorizontal": undefined,
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "paddingTop": 2,
                      "width": undefined,
                    }
                  }
                >
                  <Text
                    style={
                      Object {
                        "color": "#333333",
                        "fontSize": 16,
                        "lineHeight": 22,
                      }
                    }
                  >
                    555-876-882771
                  </Text>
                </View>
              </View>
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": undefined,
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": undefined,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": undefined,
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": 32,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#BBBBBB",
                      "fontSize": 15,
                    }
                  }
                >
                  Mobile
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>,
  <View
    pointerEvents="auto"
    style={
      Object {
        "alignItems": undefined,
        "backgroundColor": "#FFFFFF",
        "borderRadius": undefined,
        "borderTopColor": "#CCCCCC",
        "borderTopWidth": 0.5,
        "flex": undefined,
        "flexDirection": undefined,
        "height": undefined,
        "justifyContent": undefined,
        "margin": undefined,
        "marginBottom": undefined,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "padding": undefined,
        "paddingBottom": undefined,
        "paddingHorizontal": undefined,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": 15,
        "width": undefined,
      }
    }
  >
    <View
      pointerEvents="auto"
      style={
        Object {
          "alignItems": undefined,
          "backgroundColor": "#FFFFFF",
          "borderRadius": undefined,
          "flex": undefined,
          "flexDirection": undefined,
          "height": undefined,
          "justifyContent": undefined,
          "margin": undefined,
          "marginBottom": undefined,
          "marginLeft": undefined,
          "marginRight": undefined,
          "marginTop": undefined,
          "padding": undefined,
          "paddingBottom": 15,
          "paddingHorizontal": 15,
          "paddingLeft": undefined,
          "paddingRight": undefined,
          "paddingTop": undefined,
          "width": undefined,
        }
      }
    >
      <View
        pointerEvents="auto"
        style={
          Object {
            "alignItems": undefined,
            "backgroundColor": undefined,
            "borderRadius": undefined,
            "flex": undefined,
            "flexDirection": "row",
            "height": undefined,
            "justifyContent": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginTop": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingTop": undefined,
            "width": undefined,
          }
        }
      >
        <View
          pointerEvents="auto"
          style={
            Object {
              "alignItems": undefined,
              "backgroundColor": undefined,
              "borderRadius": undefined,
              "flex": 1,
              "flexDirection": undefined,
              "height": undefined,
              "justifyContent": undefined,
              "margin": undefined,
              "marginBottom": undefined,
              "marginLeft": undefined,
              "marginRight": undefined,
              "marginTop": undefined,
              "padding": undefined,
              "paddingBottom": undefined,
              "paddingHorizontal": undefined,
              "paddingLeft": undefined,
              "paddingRight": undefined,
              "paddingTop": undefined,
              "width": undefined,
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "borderRadius": 8,
                "maxWidth": 300,
                "padding": 15,
              }
            }
          >
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": undefined,
                  "backgroundColor": undefined,
                  "borderRadius": undefined,
                  "flex": undefined,
                  "flexDirection": "row",
                  "height": undefined,
                  "justifyContent": "center",
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": undefined,
                  "paddingHorizontal": undefined,
                  "paddingLeft": undefined,
                  "paddingRight": undefined,
                  "paddingTop": undefined,
                  "width": undefined,
                }
              }
            >
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": undefined,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": "center",
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#3DCF8A",
                      "fontSize": 18,
                    }
                  }
                >
                  Decline
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          pointerEvents="auto"
          style={
            Object {
              "alignItems": undefined,
              "backgroundColor": undefined,
              "borderRadius": undefined,
              "flex": 1,
              "flexDirection": undefined,
              "height": undefined,
              "justifyContent": undefined,
              "margin": undefined,
              "marginBottom": undefined,
              "marginLeft": undefined,
              "marginRight": undefined,
              "marginTop": undefined,
              "padding": undefined,
              "paddingBottom": undefined,
              "paddingHorizontal": undefined,
              "paddingLeft": undefined,
              "paddingRight": undefined,
              "paddingTop": undefined,
              "width": undefined,
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#0078ff",
                "borderRadius": 8,
                "elevation": 2,
                "maxWidth": 300,
                "padding": 15,
                "shadowColor": "#000000",
                "shadowOpacity": 0.2,
                "shadowRadius": 15,
              }
            }
          >
            <View
              pointerEvents="auto"
              style={
                Object {
                  "alignItems": undefined,
                  "backgroundColor": undefined,
                  "borderRadius": undefined,
                  "flex": undefined,
                  "flexDirection": "row",
                  "height": undefined,
                  "justifyContent": "center",
                  "margin": undefined,
                  "marginBottom": undefined,
                  "marginLeft": undefined,
                  "marginRight": undefined,
                  "marginTop": undefined,
                  "padding": undefined,
                  "paddingBottom": undefined,
                  "paddingHorizontal": undefined,
                  "paddingLeft": undefined,
                  "paddingRight": undefined,
                  "paddingTop": undefined,
                  "width": undefined,
                }
              }
            >
              <View
                pointerEvents="auto"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": undefined,
                    "borderRadius": undefined,
                    "flex": undefined,
                    "flexDirection": undefined,
                    "height": undefined,
                    "justifyContent": "center",
                    "margin": undefined,
                    "marginBottom": undefined,
                    "marginLeft": undefined,
                    "marginRight": undefined,
                    "marginTop": undefined,
                    "padding": undefined,
                    "paddingBottom": undefined,
                    "paddingHorizontal": undefined,
                    "paddingLeft": undefined,
                    "paddingRight": undefined,
                    "paddingTop": undefined,
                    "width": undefined,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#FFFFFF",
                      "fontSize": 18,
                    }
                  }
                >
                  Accept
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>,
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
      }
    }
  />,
]
`;
